rmgpy/data/kinetics/family.py

```diff
@@ -3123,7 +3123,7 @@ def extend_node(self, parent, template_rxn_map, obj=None, T=1000.0, ):
         return True
 
     def generate_tree(self, rxns=None, obj=None, thermo_database=None, T=1000.0, nprocs=1, min_splitable_entry_num=2,
-                      min_rxns_to_spawn=20, max_batch_size=800, outlier_fraction=0.02, stratum_num=8,
+                      min_rxns_to_spawn=20, max_batch_size=1000, outlier_fraction=0.02, stratum_num=8,
```

Comment:
So I chose 800 because it seemed to be the best number working with the cascade algorithm it's just nicer in some cases if the number of training reactions is just a little over 800 to just run all of them in one batch. There's probably a nice way (looking at # training reactions / max_batch_size being >1 but maybe less than 1.3) to auto detect these cases in the script.